packages/msgpck/msgpck.1.5/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+name: "msgpck"
```

Comment:
```suggestion
```
This is redundant. Opam will override it with the name obtained from the opam file path in the opam repository